{
    "apps.connection.batch_processing_try_later": "ไม่สามารถดำเนินการได้ โปรดลองอีกครั้งหลังจากนี้อีกสักครู่"
}